source/DSCResources/DSC_CMBoundaryGroups/DSC_CMBoundaryGroups.psm1

```diff
@@ -69,16 +88,34 @@ function Get-TargetResource
         Specifies the SiteCode for the Configuration Manager site.
 
     .PARAMETER BoundaryGroup
-        Specifies the boundary group name.
+        Specifies the Boundary Group name.
 
     .Parameter Boundaries
         Specifies an array of boundaries to add or remove.
 
     .Parameter BoundaryAction
-        Specifies the boundaries are to match, add, or remove Boundaries from the boundary group.
+        Specifies the boundaries are to match, add, or remove Boundaries from the Boundary Group.
```

Comment:
we should either capitalize or lowercase 'boundaries' everywhere for consistency